frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/TaskSidebarResource.tsx

```diff
@@ -17,23 +17,23 @@ type TaskSidebarResourceProps = {
 const TaskSidebarResource: React.FC<TaskSidebarResourceProps> = (props) => {
   const { availableResources, onChange, resource, taskResource } = props;
 
-  const dropdownResources = availableResources.filter(({ type }) => resource.type === type);
+  const dropdownResources = availableResources.filter(
+    ({ name, type }) => resource.type === type && !!name,
+  );
 
   return (
     <FormGroup
       fieldId={resource.name}
       label={resource.name}
       helperText={`Only showing resources for this type (${resource.type}).`}
```

Comment:
Might want to full on get rid of this. Resources apparently have [descriptions](https://github.com/tektoncd/pipeline/blob/master/pkg/apis/resource/v1alpha1/pipeline_resource_types.go#L132), it's just not described anywhere. May put this logic in this place if UX thinks this text is overkill given the update to the placeholder text.